src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -137,7 +137,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $validAssetDirs = [];
         /** @var BundleInterface $bundle */
         foreach ($kernel->getBundles() as $bundle) {
-            if (!is_dir($originDir = $bundle->getPath().'/Resources/public')) {
+            if (!method_exists($bundle, 'getPublicPath')) {
+                @trigger_error('Not definind getPublicPath() method is deprecated since Symfony 4.4 and will not be supported in 5.0.', E_USER_DEPRECATED);
+                $publicPath = '/Resources/public';
```

Comment:
If it's a relative path, I would not use `/` at the beginning of the value.

Reply:
This was the old value (I mean, the value that is currently returned in Symfony code), I think we should keep as it

Reply:
Where is it returned?

Reply:
The value currently used by Symfony for `$originDir` is `$bundle->getPath().'/Resources/public'`, as you can see above, in line 140

Reply:
Which is different, right? Here, we have a relative path, there, an absolute one.

Reply:
Sorry, I don't get you.
The old code is `$originDir = $bundle->getPath().'/Resources/public'`
The new code is `$publicPath = '/Resources/public'; $originDir = $bundle->getPath().$publicPath` (except for checking the new method, of course)
I can't see any change about relative/absolute.

Reply:
I think that the code should be `$publicPath = 'Resources/public';` and then `if (!is_dir($originDir = $bundle->getPath().'/'.$publicPath)) {`